### PR TITLE
Add IBM J9 VM to list of supported JDK

### PR DESCRIPTION
**Pull request by MishaDemianenko**
Update JvmChecker to not complain when running on IBM J9 VM.

### CODE REVIEW

**Comment by benbc** [comment]
@MishaDemianenko There are a couple more places this needs adding. It needs to be handled by the shell scripts (in `neo4j-shared.sh`) and mentioned in the docs (in `requirements.asciidoc`).

Is there now a compatibility build to verify this support?

**Comment by MishaDemianenko** [comment]
@benbc builds are our compatibility builds on IBM JDK.
Wow, tried to find all entries in java files will update those as well, ta

**Comment by MishaDemianenko** [comment]
@benbc `requirements.asciidoc` already mention IBM JDK, but still update couple of other places.
@mariascharin can you also take a look into docs part.
@srbaker there is a read my file for Debian package that mention specific JDKs for some reason, would assume that should be updated and reference supported JDKs, right?

**Comment by benbc** [comment]
@MishaDemianenko Thanks. You even found the script unit test!

File: community/server/src/docs/ops/server-installation.asciidoc [hunk 1]
```diff
@@ -18,7 +18,9 @@ For information on High Availability, please refer to <<ha>>.
 == Prerequisites ==
 
 With the exception of our Windows and Mac Installers, you'll need a Java Virtual Machine installed on your computer.
-We recommend that you install http://openjdk.java.net/[OpenJDK 8] or http://www.oracle.com/technetwork/java/javase/downloads/index.html[Oracle Java 8].
+We recommend that you install http://openjdk.java.net/[OpenJDK 8],
+http://www.oracle.com/technetwork/java/javase/downloads/index.html[Oracle Java 8] or
+http://www.ibm.com/developerworks/java/jdk/[IBM Java 8] (POWER8 only).
```
Comment (chrisvest) [inline community/server/src/docs/ops/server-installation.asciidoc:23]: We also have an x86 build with IBM JDK, though.

Comment (MishaDemianenko) [inline community/server/src/docs/ops/server-installation.asciidoc:23]: Yes have noticed it, that is something i would like to talk with @mariascharin , as soon as will get her attention :)

**Pull request merged**